sources/packages/backend/apps/queue-consumers/src/processors/assessment/_tests_/cancel-application-assessment.processor.e2e-spec.ts

```diff
@@ -154,6 +165,30 @@ describe(
       expect(result.summary).toContain("Assessment cancelled with success.");
     });
 
+    it("Should log a warning message when the assessment has status different than 'cancelation queued'.", async () => {
```

Comment:
Not a blocker but we do use some enum in the E2E description, for instance `${StudentAssessmentStatus.CancellationQueued}` instead of "cancelation queued".
```ts
`Should log a warning message when the assessment has status different than '${StudentAssessmentStatus.CancellationQueued}'.`
```

Reply:
Was this suggestion accepted? I saw the thumbs up but I am not able to see the code change.